### DISCUSSION TITLE
documentation of example compounds

### DISCUSSION THREAD

**Discussion by RKasp** [Open-Systems-Pharmacology/Forum#824] [Questions & Problems] [answered]
Hello and happy new year,
I am wondering if OSP has documentation for the older build-in example compounds such as theophylline, paracetamol, .... I am aware about the publications, but they do not cover all aspects wrt metabolism or renal excretion, what has been included in these example model files in PKSim. Any suggestion about accompanying files for explanation/documentation would be great. Thank you.
Best regards,
Rudiger

**Comment by JanSchlender** [comment, marked as answer]
Hello Rudiger,
Currently, there is only a reference to the existing publications. The modellers, who established those template, will most likely add additional information in a repository. However, these models are only reliable for the cases they were established and/or verified. Feel free to edit these models and share your experiance with the community (as done for acetaminophene). #141 
Best wishes,
Jan

**Comment by RKasp** [comment]
Hello Jan,
many thanks, this is helpful, I will look into it. Indeed having a summary sheet for a template model with some justification for certain model choices would be good.
Best regards, Rudiger
